packages/amplify-graphql-predictions-transformer/src/graphql-predictions-transformer.ts

```diff
@@ -153,6 +153,12 @@ export class PredictionsTransformer extends TransformerPluginBase {
   };
 
   generateResolvers = (context: TransformerContextProvider): void => {
+    // This validation can't occur in validate because the api has not been initialized until generateResolvers
+    if (!context.transformParameters.allowGen1Patterns) {
+      cdk.Annotations.of(context.api).addWarning(
```

Comment:
Annotations also has `addDeprecation`. I chose to use `addWarning` here because `addDeprecation` appears to be geared towards deprecating an entire parameter. We are removing some functionality from a parameter.